--- sbin/puppetd	2010-07-18 14:17:38.000000000 -0700
+++ sbin/puppetd	2010-07-21 17:31:57.057138615 -0700
@@ -1,4 +1,4 @@
-#!/usr/bin/env ruby
+#!/usr/bin/ruby
 
 require 'puppet/application/agent'
 Puppet::Application[:agent].run

--- sbin/puppetmasterd	2010-07-18 14:17:38.000000000 -0700
+++ sbin/puppetmasterd	2010-07-21 17:32:10.178513969 -0700
@@ -1,4 +1,4 @@
-#!/usr/bin/env ruby
+#!/usr/bin/ruby
 
 require 'puppet/application/master'
 Puppet::Application[:master].run
